### DISCUSSION TITLE
manual installation, without package manager

### DISCUSSION THREAD

**Discussion by v-georg-v** [saghen/blink.cmp#384] [Q&A] [answered]
Hello!

How can I install plugin without package manager?
When I download repo and locate it in specific directory, neovim can see it, but I get error:
""" 
"Can't download from github due to not being on a git tag and no fuzzy.prebuilt_binaries.force_version set, but found no built version of the library. "
              .. 'Either run `cargo build --release` via your package manager, switch to a git tag, or set `fuzzy.prebuilt_binaries.force_version` in config. '
              .. 'See the README for more info.'
"""

My OS is Linux Debian 12 and i need to download [x86_64-unknown-linux-gnu.so](https://github.com/Saghen/blink.cmp/releases/download/v0.6.1/x86_64-unknown-linux-gnu.so), I don't have rust compiler and can't install it.
What next i must do?

**Comment by saghen** [comment]
Hey there! You need to run `git checkout v0.6.1` (run `git tag` to see all available versions) in the cloned repo

**Reply by ostapkob** [reply]
If I can't use Git on this machine?

**Reply by v-georg-v** [reply]
I cannot use git tool.
Only if download.

**Comment by futsuuu** [comment, marked as answer]
I think that you need to

1. download blink.cmp v0.6.1 into `path/to/blink.cmp/` 
2. and download `x86_64-unknown-linux-gnu.so` into `path/to/blink.cmp/target/release/libblink_cmp_fuzzy.so`.

**Reply by v-georg-v** [reply]
thank you, now it is working.
